core/templates/dev/head/pages/topic_viewer/stories_list_directive.html

```diff
@@ -90,7 +90,7 @@
   stories-list div .oppia-topic-viewer-carousel {
     float: left;
     height: 430px;
-    max-width: 1070px;
+    max-width: 1420px;
```

Comment:
This seems a little big. Does it work well on mobile?

Reply:
@anubhavsinha98 Can you confirm this?

Reply:
Yeah, I just checked it on mobile view it seems its failing. Will fix it and update the PR.

Reply:
@anubhavsinha98, are we expecting this to be fixed?

Reply:
I have fixed it and actually in mobile view we get a list of story tiles vertically. Akshay has added screenshot for it. i think it can help.